### PR TITLE
Vedant's first contribution

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
I kept the PR title so because this PR does many things which cannot be described in the title.

### CODE REVIEW

**Comment by vedantmgoyal9** [comment]
@jedieaston Would you like to give it a go?

**Comment by jedieaston** [comment]
I can attempt to review it sometime this week, but can you summarize what you did? Since I don't have a test suite I can't automatically review your changes to make sure functionality wasn't broken.

**Comment by vedantmgoyal9** [comment]
- Update documentation (README.md)
- Add CODEONWERS so that we don't need to comment "@jedieaston please review/take a look upon it" and can directly request a review from you
- Add PSScriptAnalyser to ensure code quality
- Add GitHub Pages theme "Cayman" because it makes documentation easier to read.
- OTBS indentation for psm1 and psd1 files

**Comment by vedantmgoyal9** [comment]
@jedieaston just writing this comment to remind you that you've to review it.

**Comment by vedantmgoyal9** [comment]
@jedieaston Is there anything I've done wrong? I don't think I've made any BREAKING changes.

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-cayman
```
Comment (jedieaston) [inline _config.yml]: I'm not a huge fan of this theme, and I'm not quite sure why it's needed? Pages isn't configured for this repository.

File: WinGetHelpers.psm1 [hunk 2]
```diff
@@ -749,61 +726,59 @@ function Convert-WinGetSingletonToMultiFile {
...
     $newManifest["installer"] = [Ordered]@{
         PackageIdentifier = $currentManifest["PackageIdentifier"];
-        PackageVersion = $currentManifest["PackageVersion"];
-        Installers = $currentManifest["Installers"];
-        ManifestType = "installer";
-        ManifestVersion = "1.0.0";
+        PackageVersion    = $currentManifest["PackageVersion"];
+        Installers        = $currentManifest["Installers"];
+        ManifestType      = "installer";
+        ManifestVersion   = "1.0.0";
```
Comment (jedieaston) [inline WinGetHelpers.psm1:750]: Why so much spacing? I don't know how much having the equal signs being in parallel is important (and I'm not sure I'll be paying attention when changing this code later).

File: .github/workflows/powershell-analysis.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+#
+# https://github.com/microsoft/action-psscriptanalyzer
+# For more information on PSScriptAnalyzer in general, see
+# https://github.com/PowerShell/PSScriptAnalyzer
+
+name: PSScriptAnalyzer
```
Comment (jedieaston) [inline .github/workflows/powershell-analysis.yml]: I don't know why we need this. I understand what ScriptAnalyzer does, but I don't feel like troubleshooting it if it blows up. 

If someone's using this for a production level service and feels like there are issues with the PowerShell scripts that ScriptAnalyzer would find, they should probably rewrite it themselves. I wrote this to get a better understanding of PoSH, not to be a battle-hardened super secure way of updating manifests (that's `wingetcreate`'s job!).

File: README.md [hunk 4]
```diff
@@ -1,16 +1,32 @@
 # WinGetHelpers!
 
-__Now updated for 1.0 manifests!__
```
Comment (jedieaston) [inline README.md:3]: Please put this message back.

File: README.md [hunk 5]
```diff
@@ -1,16 +1,32 @@
...
+This is a PowerShell module that contains a collection of helper functions to make creation/submission of manifests easy for [microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs).
 
+## Dependencies
 
-These are some cmdlets I hastely made to help support microsoft/winget-pkgs. Maybe they'll help you too.
```
Comment (jedieaston) [inline README.md:6]: Please put this message back.

File: README.md [hunk 6]
```diff
@@ -1,16 +1,32 @@
...
-# Requirements
+You can install both of these at once with the following command:
+```
+Install-Module powershell-yaml; Install-Module -Name Carbon
+```
 
-You'll need two dependencies: [Carbon](http://get-carbon.org/) (for getting the product code from MSI files) and [powershell-yaml](https://github.com/cloudbase/powershell-yaml) (to read and write YAML files). 
+## Cmdlets/Functions in this module
 
-So before you download this module and put it in your modules folder, make sure to do ` Install-Module powershell-yaml; Install-Module -Name 'Carbon' `
```
Comment (jedieaston) [inline README.md:15]: Please put this message back. People who want to use this may not be familiar with PowerShell.

File: README.md [hunk 7]
```diff
@@ -1,16 +1,32 @@
...
 
-__Now updated for 1.0 manifests!__
+This is a PowerShell module that contains a collection of helper functions to make creation/submission of manifests easy for [microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs).
 
+## Dependencies
 
-These are some cmdlets I hastely made to help support microsoft/winget-pkgs. Maybe they'll help you too. 
+This PowerShell module requires the following dependencies:
 
-I'm not very good at PowerShell, which is why it looks so bad, but all of these work on my machine(tm) so as long as you are on Windows 10 and have winget installed,
```
Comment (jedieaston) [inline README.md:8]: Please put this message back.

**Review by jedieaston — CHANGES_REQUESTED** [review]
I reviewed it, much of it seems fine but somewhat unnecessary (that's a lot of spacing). If you make the suggested changes I'll approve this.

File: README.md (hunk 6 above)
Comment (vedantmgoyal2009) [reply README.md:15]: I wrote something similar here.

https://github.com/vedantmgoyal2009/WinGetHelpers/blob/patch-1/README.md?plain=1#L14-L17

File: WinGetHelpers.psm1 (hunk 2 above)
Comment (vedantmgoyal9) [reply WinGetHelpers.psm1:750]: That's a part of OTBS indentation. I didn't do it manually. VS Code automatically did it.

File: README.md (hunk 6 above)
Comment (vedantmgoyal2009) [reply README.md:15]: https://github.com/jedieaston/WinGetHelpers/pull/4/commits/78f3a48c1144055b2095b5d0695ab9d05998fe8a#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R18-R21

Comment (vedantmgoyal9) [reply README.md:15]: https://github.com/vedantmgoyal2009/WinGetHelpers/blob/patch-1/README.md?plain=1#L18-L21

Comment (vedantmgoyal2009) [reply README.md:15]: Please refer to the last one. The above two are outdated.

**Comment by vedantmgoyal9** [comment]
@jedieaston I've made the changes to the documentation. However, I have changed the language a bit.

File: README.md [hunk 8]
```diff
@@ -1,16 +1,36 @@
 # WinGetHelpers!
 
-__Now updated for 1.0 manifests!__
+__Now updated for 1.0 manifests__
 
+I hastily made this PowerShell module, that contains a collection of helper functions to make creation/submission of manifests easy for [microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs). Maybe it will be useful for other people too.
```
Comment (jedieaston) [inline README.md]:
```suggestion
I hastily made this PowerShell module, that contains a collection of helper functions to make creation/submission of manifests easy for [microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs). Maybe it will be useful for you too.
```

File: README.md [hunk 9]
```diff
@@ -1,16 +1,36 @@
 # WinGetHelpers!
 
-__Now updated for 1.0 manifests!__
+__Now updated for 1.0 manifests__
```
Comment (jedieaston) [inline README.md]:
```suggestion
__Now updated for 1.0 manifests!__
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
two quick things and I'll merge.

**Review by jedieaston — APPROVED** [review]

**Comment by vedantmgoyal2009** [comment]
@jedieaston please accept the invitation on LinkedIn 🙏🏻
